ros/src/msgs/autoware_msgs/msg/TwistGate.msg

```diff
@@ -0,0 +1,12 @@
+Header header
+int32 accel
+int32 brake
+int32 steer
+int32 gear
+int32 blinker
+int32 mode
+int32 emergency
+float64 linear_x
```

Comment:
Use autoware_msgs like above comment and use geometry_msgs::Twist for linear_x and angular_z.

Reply:
How do you use linear_x and linear_velocity ? If they have different meanings, please change to clear names.